requirements/base.txt

```diff
@@ -16,3 +16,7 @@ requests==2.12.3          # via slumber
 six==1.10.0               # via django-extensions
 slumber==0.7.1            # via edx-rest-api-client
 unicodecsv==0.14.1
+djangorestframework==3.2.3
```

Comment:
Maybe I'm not really understanding how dependency resolution works here, but in edx-platform the version of Django is pinned to [1.8.17](https://github.com/edx/edx-platform/blame/master/requirements/edx/base.txt#L41), so it seems odd that we would be able to specify 1.10.3 for this package and expect it to work.

I'm wondering if it's possible to downgrade the Django dependency here to 1.8.17 in order to address the DRF 3.2.3 conflict, or if doing that would cause issues with some of the other Django-based features we are utilizing, such as some of the advanced Admin tooling.  Or perhaps downgrading Django is not the right solution at all?  I don't see us bumping edx-platform to 1.10+ any time soon.

@bradenmacdonald @haikuginger @e-kolpakov would be great to get your take on this.